### DISCUSSION TITLE
Entry point?

### DISCUSSION THREAD

**Discussion by henryiii** [Textualize/rich#898] [Ideas]
I assume it was discussed somewhere, but didn't see it, but why is there no entry point for rich? Thinks like `rich.markdown` (wow) and others seem like they would be great as a shell entry point / entry-points, so rich could be installed directly to the system with `pipx`. `rich markdown` or `rich-markdown` would be very nice to have everywhere, I think. Maybe `rich tree/rich-tree`, etc. Might start becoming a goto along the lines of [bat]([link=URL]text[/link]), possibly.

**Comment by willmcgugan** [comment]
TBH I never thought of the rich.markdown or rich.syntax commands to be anything other than demonstrations. Trouble is, if I push them as a useful tool I'm going to get asked for more features and fixes. And spare time can only stretch so far.

**Reply by henryiii** [reply]
I feel like you are almost all the way there, and you'd not have to advertise it too heavily if you didn't want to. But understand about free time! 🤦
